code/game/objects/items/cardboard_cutouts.dm

```diff
@@ -0,0 +1,175 @@
+//Cardboard cutouts! They're man-shaped and can be colored with a crayon to look like a human in a certain outfit, although it's limited, discolored, and obvious to more than a cursory glance.
+/obj/item/cardboard_cutout
+	name = "cardboard cutout"
+	desc = "A vaguely humanoid cardboard cutout. It's completely blank."
+	icon = 'icons/obj/cardboard_cutout.dmi'
+	icon_state = "cutout_basic"
+	burn_state = FLAMMABLE
+	w_class = 4
+	var/list/possible_appearances = list("Assistant", "Clown", "Mime",
+		"Traitor", "Nuke Op", "Cultist", "Clockwork Cultist",
+		"Revolutionary", "Wizard", "Shadowling", "Xenomorph", "Swarmer",
+		"Ash Walker", "Deathsquad Officer", "Ian", "Slaughter Demon",
+		"Laughter Demon", "Xenomorph Maid", "Private Security Officer")
```

Comment:
With the... arguements... lately over Xenomorph maid, maybe we don't include this one?